.github/COMMIT_CONVENTION.md

```diff
@@ -0,0 +1,102 @@
+## Git Commit Message Convention
+
+> This is adapted from [Angular's commit convention](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular).
+
+#### TL;DR:
+
+Messages must be matched by the following regex:
+
+``` js
+/^(revert: )?(feat|fix|docs|style|refactor|test|ci|chore)(\(.+\))?: .{1,50}/
```

Comment:
50 chars max?

This won't match: `feat(foo): add something to the user account form to make it work properly`

Reply:
> 50 chars max?

Yep.

> This won't match: feat(foo): add something to the user account form to make it work properly

And?

Reply:
let's make 70

Reply:
70